subprojects/deploy-cluster/config_install.mk

```diff
@@ -0,0 +1,87 @@
+# https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-install-linux.tar.gz"
+# https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp-dev-preview/pre-release/openshift-install-linux.tar.gz"
+
+has_installer: ${OPENSHIFT_INSTALLER}
+${OPENSHIFT_INSTALLER}:
+	@echo "WARNING: Installer v${OCP_VERSION} not found: ${OPENSHIFT_INSTALLER}"
+	@echo "Downloading it from: "
+	mkdir -p $$(dirname  "${OPENSHIFT_INSTALLER}")
+	wget "${OPENSHIFT_INSTALLER_URL}" && set -x \
+        && tar xzf openshift-install-linux-${OCP_VERSION}.tar.gz openshift-install \
+	&& mv openshift-install "${OPENSHIFT_INSTALLER}" \
+	&& rm openshift-install-linux-${OCP_VERSION}.tar.gz
+	test -e "${OPENSHIFT_INSTALLER}"
+
+manifest: has_installer
+	@[[ -f ${CLUSTER_PATH}/install-config.yaml ]] && cp ${CLUSTER_PATH}/install-config{,.back}.yaml || echo
+	${OPENSHIFT_INSTALLER} create manifests --dir=${CLUSTER_PATH} > /dev/null;
+
+install: has_installer
+	@[[ -f ${CLUSTER_PATH}/install-config.yaml ]] && cp ${CLUSTER_PATH}/install-config{,.back}.yaml || echo
+	@if [ -e ${CLUSTER_PATH}/install.log ]; then \
+	  echo "INFO: Found ${CLUSTER_PATH}/install.log" \
+	  echo "ERROR: Cluster already installed ..."; \
+	  exit 1; \
+	fi;
+	time ${OPENSHIFT_INSTALLER} create cluster --dir=${CLUSTER_PATH} --log-level=debug 2>&1 | tee ${CLUSTER_PATH}/install.log
+
+config_new_install: has_installer
+	@mkdir ${CLUSTER_PATH} -p
+	@if [ -f '${INSTALL_CONFIG}' ]; then\
```

Comment:
I'm confused about the inconsistent use of `[[` and `[` any particular reason for it?